vscode/src/projectSystem.ts

```diff
@@ -1,10 +1,17 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { log } from "qsharp-lang";
 import { Utils, URI } from "vscode-uri";
 import * as vscode from "vscode";
 
+import { getProjectLoader, log } from "qsharp-lang";
+
+export async function isQsharpProject(uri: vscode.Uri): Promise<boolean> {
+  // we look for a manifest file in the parent directory of the current file
```

Comment:
This comment is a bit misleading since it implies we only look in the _parent_ directory

```suggestion
  // we look for the manifest in the directory of the current file and all of its parent directories
```